src/Microsoft.ML.TorchSharp/NasBert/TaskType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.TorchSharp.NasBert
+{
+    public enum TaskType
+    {
+        None = 0,
+        MaskedLM = 1,
```

Comment:
>MaskedLM [](http://example.com/codeflow?start=8&length=8)

are we picking these terms from the Bert model? would be good if at least have a small comment about such terms here.

Reply:
Good point. Added that in.